app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -87,6 +110,18 @@ A filename. This is the main entrypoint into your application for all routes. Of
 your `index.html` file. This file should import the app shell file specified in the
 [`shell`](#shell) option. It should be minimal since it's loaded and cached for each route.
 
+### --fragment {#fragment}
+
+This flag supports dynamic dependencies. It is an array of any HTML filenames that are not
+statically linked from the app shell (that is, imports loaded on demand by `importHref`).
```

Comment:
I think `importHref` has been replaced with dynamic import (`import(url).then(...)`). Also, @usergenic mentioned to me the other day that build can detect dynamic dependencies automatically iff the url passed to `import()` is a static string.

Reply:
Also, this section still talks about these imports as if they were HTML imports; maybe I'm missing something about what this field does?

Reply:
>this section still talks about these imports as if they were HTML imports; maybe I'm missing something about what this field does?

nah i just missed this bit :D thanks

Reply:
@usergenic can you clarify how the `--fragment` flag and `fragments` array in Polymer.json currently work?

Reply:
They're the same as before, but now JS files to be loaded by dynamic `import()` (or any other means really, like a `<script>` tag.

Reply:
Since `import('./path.js')` is automatically detected now, we don't use the `fragments` config any more in 3.0 apps.